build/lin.sh

```diff
@@ -158,7 +158,7 @@ version_latest "orc" "$VERSION_ORC" "2573"
 version_latest "proxy-libintl" "$VERSION_PROXY_LIBINTL" "frida/proxy-libintl"
 version_latest "gdkpixbuf" "$VERSION_GDKPIXBUF" "9533"
 version_latest "freetype" "$VERSION_FREETYPE" "854"
-version_latest "expat" "$VERSION_EXPAT" "770"
+#version_latest "expat" "$VERSION_EXPAT" "770" # latest version in release monitoring is out of sync
```

Comment:
We could use a GitHub tag check here instead:
```suggestion
version_latest "expat" "$VERSION_EXPAT" "770" "libexpat/libexpat"
```

Reply:
I think that needs more modifications since the upstream repository uses a unusual tag convention, see e.g.:
https://github.com/libexpat/libexpat/releases/tag/R_2_4_9

Reply:
Ah, I see what you mean, please ignore this suggestion.